.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+<!-- This is a template - do add to or remove from as needed -->
+<!-- Please provide a general summary of your changes in the Title above 🚀 -->
+
+## Description 💬
+<!-- Please describe your changes in detail -->
+
+## How Has This Been Tested? 🖥️
+[Preview](https://<branch>--splitify-preview.netlify.app)
+
+## Checklist ✅:
+<!-- Go over all the following points, and check all the boxes. -->
+- [ ] Added a description of my changes
+- [ ] I assigned the story to myself on pivotal tracker before I started working on this
+- [ ] Added details of how I tested my changes
+- [ ] Replaced <branch> with the branch we're merging from
```

Comment:
* Add user story link
* Netlify will automatically build and post a link to the preview of the current PR. I reckon branch deploy previews aren't critical to include as part of the PR

```suggestion
## Checklist ✅:
<!-- Go over all the following points, and check all the boxes. -->
- [ ] I assigned the story to myself on Pivotal Tracker before I started working on this
- [ ] Added a link to the user story context
- [ ] Added a description of my changes
```